scripts/pytest-utils.sh

```
@@ -42,3 +42,44 @@ pytest() {
 
     python3 -u -m pytest "${pytest_extra_args[@]}" "$@" || $TRITON_TEST_IGNORE_ERRORS
 }
+
+capture_runtime_env() {
+    mkdir -p "$TRITON_TEST_REPORTS_DIR"
+
+    echo "$CMPLR_ROOT" > $TRITON_TEST_REPORTS_DIR/cmplr_version.txt
+    echo "$MKLROOT" > $TRITON_TEST_REPORTS_DIR/mkl_version.txt
+
+    if [ -d "${SCRIPTS_DIR}/../python/dist" ]; then
+        WHEEL=$(find ${SCRIPTS_DIR}/../python/dist -type f -name *.whl)
+        if [[ ! -z "$WHEEL" ]]; then
+            echo "$WHEEL" | sed -n 's/.*git\([a-zA-Z0-9]*\)[^a-zA-Z0-9].*/\1/p' > $TRITON_TEST_REPORTS_DIR/triton_commit_id.txt
+        else
+            echo "Triton wheel is not found" > $TRITON_TEST_REPORTS_DIR/triton_commit_id.txt
+        fi
+    else
+        echo "Triton wheel directory is not found" > $TRITON_TEST_REPORTS_DIR/triton_commit_id.txt
```

I think the script needs to fail rather than print this to `triton_commit_id.txt`.

---

done ~